samples/enable_pubsub_notifications.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# [START contactcenterinsights_enable_pubsub_notifications]
+from google.cloud.contact_center_insights_v1.services.contact_center_insights import client
+from google.cloud.contact_center_insights_v1.types import resources
+from google.protobuf import field_mask_pb2
+
+
+def enable_pubsub_notifications(project_id: str, topic_create_conversation: str,
+                                topic_create_analysis: str) -> None:
+    # Construct a settings resource.
+    settings = resources.Settings()
+    settings.name = client.ContactCenterInsightsClient.settings_path(project_id, 'us-central1')
```

Comment:
Thank you for using the resource path helper method!

Reply:
I found this helper to be super useful!